dot/core/service_test.go

```diff
@@ -28,6 +36,78 @@ import (
 
 var errTestDummyError = errors.New("test dummy error")
 
+func generateTestCentrifugeMetadata(t *testing.T) *ctypes.Metadata {
+	t.Helper()
+	rawMeta := common.MustHexToBytes(testdata.NewTestMetadata())
```

Comment:
nit let's avoid panicing in tests

```suggestion

	metadataHex := testdata.NewTestMetadata()
	rawMeta, err := common.HexToBytes(metadataHex)
	require.NoError(t, err)

```